### ISSUE TITLE
Extracted counted word list (FIO)

### ISSUE THREAD

**Issue by DavidHaslam** [tfbf/punjabi_bible_1945#43]
After some of my provisional corrections, I used a **TextPipe** filter to extract and count all the Gurmukhi words found in verse text in my concatenated USFM file.

[merged.words.count.txt](https://github.com/tfbf/punjabi_bible_1945/files/674360/merged.words.count.txt)

The file is tab delimited and sorted on the second field.
Examination of words that are found only once (_hapax legomena_) may give clues to any unexpected spellings.

Clearly this exercise will require knowledge of Punjabi and the Biblical context.

All punctuation, etc. (including the DANDA) was removed before the word list was generated.
The list excludes words found in section titles and after other non-verse markers.

Careful examination with a suitable Unicode text editor should also expose the anomalous double vowel signs and the like that were reported separately earlier today.

Tip: The file could readily be opened with Microsoft Excel such that an advanced user might make use of the **Data | Filter | Autofilter** features in Excel to pursue the detailed investigation.

**Comment by DavidHaslam** [comment]
(FIO) Character frequency analysis of the counted word list:

[merged.words.count.character.frequency.txt](https://github.com/tfbf/punjabi_bible_1945/files/674362/merged.words.count.character.frequency.txt)

**Comment by DavidHaslam** [comment]
Illustration of how to use this list.

Four consecutive lines look similar:

000001	ਡਗਮਗਾਉਂਦਾ
000001	ਡਗਮਗਾਉਂਦੀ
000004	ਡਗਮਗਾਉਂਦੇ
000003	ਡਗਮਗਾਉਣ

Google Translate gave:

000001 dagamagaunda
000001 moves unsteadily
000004 dagamagaunde
000003 dagamagauna

Only one of these words is a translation. The other three are merely transliterations.
Those three may contain typographical errors. The Punjabi expert should decide.

Have fun!

**Comment by DavidHaslam** [comment]
Aside: **TextPipe Standard** is not free software. It's available from [DataMystic](http://www.datamystic.com/textpipe/standard). 
I've been a user for 15 years.

**Comment by DavidHaslam** [comment]
Aside: Even the concatenation of the 66 USFM files is something I use TextPipe for.
I have this filter integrated into my **Windows Shell** extensions.

**Comment by DavidHaslam** [comment]
My TextPipe filter to extract and count the Gurmukhi words has been subsequently enhanced to provide a secondary output of counted **glyphs**. 

See https://github.com/tfbf/punjabi_bible_1945/issues/44

The filtered 99 unique instances of **bad signs** are listed in the attached text file:

[merged.glyphs.count.bad.signs.txt](https://github.com/tfbf/punjabi_bible_1945/files/677512/merged.glyphs.count.bad.signs.txt)

Best viewed using BabelPad.

**Comment by DavidHaslam** [comment]
Superseded by more recent analysis.
